packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/allocate_disk_space_page.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:filesize/filesize.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_spinbox/material.dart';
```

Comment:
Shouldn't that import be in `allocate_disk_space_dialogs.dart`, where a `SpinBox` is instantiated?

Reply:
`_dialogs.dart` and `_widgets.dart` are `part of` `_page.dart`.

it's a trick to be able to keep the widgets private and avoid documenting them. i'll make them standalone files instead... :P